ConfigSamples/Smoothieboard/config

```diff
@@ -298,6 +298,8 @@ panel.gamma_jog_feedrate                     200               # z jogging feedr
 panel.hotend_temperature                     185               # temp to set hotend when preheat is selected
 panel.bed_temperature                        60                # temp to set bed when preheat is selected
 
+#panel.root_path                             /sd/gcode         # Only show this path and below in the Play menu
```

Comment:
do not put this in the config it should sijmply be documented in the wiki or maybe in the snippets.